squid: change ksh reference in krb ldap helper to sh

Very simple cert_tool script with no ksh specifics.
Change it to use sh so a package dependency doesn't
get created to ksh.

Upstream-Status: Pending

Signed-off-by: Jim Somerville <Jim.Somerville@windriver.com>
Signed-off-by: Chong Lu <Chong.Lu@windriver.com>
---
 helpers/external_acl/kerberos_ldap_group/cert_tool | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/helpers/external_acl/kerberos_ldap_group/cert_tool b/helpers/external_acl/kerberos_ldap_group/cert_tool
index 9f14959..8c4ea11 100644
--- a/helpers/external_acl/kerberos_ldap_group/cert_tool
+++ b/helpers/external_acl/kerberos_ldap_group/cert_tool
@@ -1,4 +1,4 @@
-#!/bin/ksh
+#!/bin/sh
 #
 #  -----------------------------------------------------------------------------
 # 
@@ -64,9 +64,9 @@ QUIT
 # Create database for Sun ldap and pem file for Openldap 
 #
 rm ${server}_[0-9]*.pem 2>/dev/null
-let i=0
+i=0
  ls ${server}_[0-9]*.cert | while read file; do
- let i=i+1
+ i=`expr $i + 1`
  cat  $file  >> ${server}_$i.pem
  CA=`openssl x509 -noout -text -in  ${server}_$i.pem | grep -i "CA:.*true"`
  if [ -n "$CA" ]; then
-- 
1.9.1

